src/layout/cursor.rs

```diff
@@ -5,15 +5,24 @@ use super::*;
 /// Represents a position within a layout.
 #[derive(Copy, Clone, Default, Debug)]
 pub struct Cursor {
-    path: CursorPath,
-    baseline: f32,
-    offset: f32,
-    advance: f32,
-    text_start: usize,
-    text_end: usize,
-    is_rtl: bool,
-    is_leading: bool,
-    is_inside: bool,
+    /// path to the target cluster
```

Comment:
Admittedly a nit, but for public docs it’d be nice if they started with a capital and ended with a period. This just feels more polished to me when reading rustdocs.

Reply:
Fair enough, I'll make that change.